ring/ring.go

```diff
@@ -265,6 +265,9 @@ type Ring struct {
 	// to be sorted alphabetically.
 	ringZones []string
 
+	// List of all ring zones ever discovered, even if they have no instances.
```

Comment:
The current is currently misleading.
```suggestion
	// Map of all ring zones ever discovered, even if they have no instances.
```